spec/requests/v0/lgy_coe_request_spec.rb

```diff
@@ -139,6 +148,34 @@
           expect(response.body).to eq @res.body
         end
       end
+
+      context 'requested document id not associated with user' do
+        before do
+          lgy_documents_response_body = [{
+            'id' => 23_929_115,
+            'document_type' => '252',
+            'create_date' => 1_670_530_715_000,
+            'description' => '[ATTACHMENT]',
+            'mime_type' => '[ATTACHMENT] example.png'
+          }, {
+            'id' => 10_101_010,
+            'document_type' => '705',
+            'create_date' => 1_670_530_714_000,
+            'description' => nil,
+            'mime_type' => 'COE Application First Returned.pdf'
+          }]
+          lgy_documents_response = double(:lgy_documents_response, body: lgy_documents_response_body)
+          expect_any_instance_of(LGY::Service).to receive(:get_coe_documents).and_return(lgy_documents_response)
+        end
+
+        it '404s' do
+          # Note that this ID is not present in lgy_documents_response_body above.
+          get '/v0/coe/document_download/12341234'
+          expect(response).to have_http_status(:not_found)
+          expect(response.content_type).to eq('text/plain; charset=utf-8')
```

Comment:
why is it not returning JSON? any reason it is plain text? Most other services and our api stuff return json stuffs